src/main/java/org/wso2/carbon/module/swiftiso20022/validation/common/ValidationRule.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.module.swiftiso20022.validation.common;
+
+import org.json.JSONObject;
+
+/**
+ * Abstract Validation Rule.
+ */
+public abstract class ValidationRule {
+
+    /**
+     * Run validation rules for given context.
```

Comment:
Let's update this comment and shall we add payload details to the comment too?

Reply:
Fixed in https://github.com/wso2-extensions/mediation-swift-iso20022-module/pull/6/commits/7f2058c938da71ec80755d296fd2d3d67541e678